### ISSUE TITLE
Not adding buildConfig to log (ACRA + Proguard)

### ISSUE THREAD

**Issue by PaijoRX** [ACRA/acra#340]
i'm just tested ACRA with enable proguard.
this is my configuration.

<b>build.gradle:</b> 
```
buildTypes {
        release {
            minifyEnabled true
            shrinkResources true
            proguardFiles getDefaultProguardFile('proguard-android.txt'), 'proguard-rules.pro'
        }
        debug{
            minifyEnabled false
            shrinkResources true
        }
    }
```
<b>proguard-rules.pro:</b>
```
-keepattributes SourceFile,LineNumberTable
-keepattributes *Annotation*
-keep class org.acra.** { *; }
```

source: https://github.com/ACRA/acra/wiki/ProGuard

and i got error message like this:
<code>
12-17 16:31:23.670  12564-12564/com.xxx.testacra E/ACRA﹕ Not adding buildConfig to log. Class Not found : com.xxx.testacra.application.BuildConfig. Please configure 'buildConfigClass' in your ACRA config
</code>

there is any mistake of my configuration?

**Comment by milesich** [comment]
This could be relevant to your problem https://github.com/ACRA/acra/pull/233

**Comment by PaijoRX** [comment]
so, what the impact of it for my application and acra process ?

**Comment by william-ferguson-au** [comment]
NB this has nothing to do with Proguard.

When ACRA is creating the crash report it will attempt to include the details of the BuildConfig (if that ReportField has been specified). It will first look in the 'buildConfigClass' parameter, if that is not specified then it will look for it in
`final String className = context.getClass().getPackage().getName() + ".BuildConfig";`

In your case it didn't find it in either.

What is the package and name of your BuildConfig class?
What version of ACRA?

**Comment by william-ferguson-au** [comment]
This was fixed in #359 as part of ACRA 4.8.1
